api/docker-compose.yaml

```diff
@@ -9,10 +9,13 @@ services:
   api:
     build:
       context: ./src/dev/server
+    user: ${UID}
     working_dir: /w
+    depends_on:
+      - db
```

Comment:
I believe this would make the database run when we run unit tests, which isn't usually necessary.

Reply:
Testing yesterday, I couldn't access the db via the `db` hostname without this. Will try getting rid of it again and document behavior if it still fails :-)

Reply:
can't reproduce, so I'll strip these as you've advised @dmohs